DEVELOPMENT.rst

```diff
@@ -25,6 +25,8 @@ For Ubuntu 16.04 e. g. follow `Yarn installation instructions
 
     $ sudo apt-get install git nodejs nodejs-legacy npm build-essential python3-dev
 
+TODO: Please check, if this is still valid for the new client
```

Comment:
Please open an issue for this and remove the TODO here.

Reply:
I would say we open one issue to fix the README and DEVELOPMENT file before the 3.0 release.